lib/dolly/request.rb

```diff
@@ -101,6 +102,16 @@ def full_path resource
     def attachment_path resource, attachment_name
       "#{full_path(resource)}/#{attachment_name}"
     end
+
+    def log_request resource, response_code
+      log_value = ->(resource, response_code) { "Query: #{resource}, Response Code: #{response_code}" }
+      case response_code
+      when 200..399
```

Comment:
I think this would be going to far for this PR, But in addition to the logging, we can add more config options so that users can config what response codes they want to log, and make it a bit more fine tuned.